lib/node/Type.js

```diff
@@ -96,5 +96,32 @@ NodeType.isIntegerFraction = function(node, allowUnaryMinus=false) {
           Number.isInteger(parseFloat(denominator.value)));
 };
 
+NodeType.isFraction = function(node, allowUnaryMinus=true, allowParens=true) {
```

Comment:
oo that's a good question. I can't think of anything off the top of my head, but if you come across something later we could generalize them together. I think this is fine for now

I could imagine something general that takes in a function that returns true for the base condition (like the `isOperator('/')` here) and does the unaryMinus/Parenthesis checking more generally no matter what the base case is

... does that make sense? We could wait until it comes up to generalize though